src/log-tlslog.c

```diff
@@ -307,10 +308,17 @@ static int LogTlsLogger(ThreadVars *tv, void *thread_data, const Packet *p,
 
     MemBufferReset(aft->buffer);
     MemBufferWriteString(aft->buffer,
-                         "%s %s:%d -> %s:%d  TLS: Subject='%s' Issuerdn='%s'",
-                         timebuf, srcip, sp, dstip, dp,
-                         ssl_state->server_connp.cert0_subject,
-                         ssl_state->server_connp.cert0_issuerdn);
+                         "%s %s:%d -> %s:%d  TLS: ",
+                         timebuf, srcip, sp, dstip, dp);
+
+    if ((ssl_state->flags & SSL_AL_FLAG_SESSION_RESUMED) != 0) {
+        MemBufferWriteString(aft->buffer, "Session resumed");
```

Comment:
Perhaps 'Session' instead. I see that it's a mix of both all-caps tokens (SNI, VERSION, NOTBEFORE, NOTAFTER) and not (Subject, Issuerdn).